galaxy_ng/app/api/ui/viewsets/execution_environment.py

```diff
@@ -97,6 +115,18 @@ def get_distro(self):
             models.ContainerDistribution, base_path=self.kwargs["base_path"])
 
 
+class ContainerTagViewset(ContainerContentBaseViewset):
+    permission_classes = [access_policy.ContainerRepositoryAccessPolicy]
+    serializer_class = serializers.ContainerTagSerializer
+    filter_backends = (DjangoFilterBackend,)
+    filterset_class = TagFilter
+
+    def get_queryset(self):
```

Comment:
Line 75 said 
```
# Tag filters are supported, but are done in get_queryset. See the comment
        # there
```

is it supposed to have a comment here?

Reply:
Ah, that refers to a different endpoint. Let me clarify the comments.